src/main/java/de/tum/cit/aet/artemis/core/security/RateLimitType.java

```diff
@@ -0,0 +1,35 @@
+package de.tum.cit.aet.artemis.core.security;
+
+/**
+ * Enum defining different rate limit types with their default requests per minute (RPM) values.
+ * These values can be overridden through application configuration.
+ */
+public enum RateLimitType {
+
+    /**
+     * Rate limit for public endpoints that don't require authentication.
+     * Default: 5 requests per minute per client.
+     */
+    PUBLIC(5),
```

Comment:
this might be too small when used for all kinds of public requests (e.g. info / health / time / index.html).
If it's only used for a subset (e.g. register), then we should rename it